src/Redux/actions.tsx

```diff
@@ -301,6 +301,16 @@ export const updateConsultationBed = (
     }
   );
 
+export const togglePatientPrivacy = (external_id: string) =>
+  fireRequest(
+    "togglePatientPrivacy",
+    [],
+    {},
+    {
+      external_id: external_id,
+    }
+  );
```

Comment:
It'd be nice to have this converted to use `request` instead of a `fireRequest` action.

(This could be done in a later PR too)

Reply:
let's do this in a later PR, there are some other minor changes as well